app/models/my_academics/semesters.rb

```diff
@@ -211,7 +211,7 @@ def add_section_grade_option(section)
 
     def add_reserved_seating_rules_link(term_id, course, section)
       if section[:waitlisted] && section[:is_primary_section]
-        reserved_capacity_count = EdoOracle::Queries.section_reserved_capacity_count(term_id, section[:ccn]).first['reserved_seating_rules_count'].to_i
+        reserved_capacity_count = EdoOracle::Queries.section_reserved_capacity_count(term_id, section[:ccn]).first.try([],'reserved_seating_rules_count').to_i
```

Comment:
@nkutub - Shouldn't it be `.try(:[], 'reserved_seating_rules_count')`

It looks like you're missing the colon before `[]`.

Reply:
opps